### PR TITLE
chore: setup build environment

### PR DESCRIPTION
**Pull request by stuart-warren**

### CODE REVIEW

**Review by Godley — APPROVED** [review]
LGTM - feels like we should have a central docker build script for travis rather than duplicating it everywhere, but meh.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+- '3.6'
+sudo: required
+services:
+- docker
+env:
+  global:
+  - REGISTRY_USER=ocadotechnologygitlab
+  - VCS_SOURCE="https://github.com/${TRAVIS_REPO_SLUG}"
```
Comment (mikebryant) [inline .travis.yml:10]: Needs `REGISTRY_PASS` set here as a `secure` variable

**Review by mikebryant — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (Godley) [reply .travis.yml:10]: Nek put the password in travis-ci directly on the mirror-operator. We should stick with one route or the other I guess.

Comment (mikebryant) [reply .travis.yml:10]: In the repo, that's what we're doing with all the other open source projects

Comment (stuart-warren) [reply .travis.yml:10]: think this is now done

**Review by mikebryant — APPROVED** [review]

**Pull request merged**